pkg/kubelet/kubelet_node_status.go

```diff
@@ -501,8 +510,13 @@ func (kl *Kubelet) setNodeStatusImages(node *api.Node) {
 		}
 
 		for _, image := range containerImages {
+			names := append(image.RepoDigests, image.RepoTags...)
```

Comment:
There is no actual "name" defined in docker images.. Digest is a content-addressable identifier which users can use to pull an image (which we also support in k8s), so it should be a legit "name" in our definition. I don't think there should be multiple digests per image (since it's sha256 hash based on the content), so I changed the order of the digests and tags to ensure that we always include the digest.